src/vm/i386/umthunkstub.S

```diff
@@ -24,7 +24,7 @@ NESTED_END TheUMEntryPrestub, _TEXT
 NESTED_ENTRY UMThunkStub, _TEXT, UnhandledExceptionHandlerUnix
 
 #define UMThunkStub_SAVEDREG                (3*4)   // ebx, esi, edi
-#define UMThunkStub_LOCALVARS               (2*4)   // UMEntryThunk*, Thread*
+#define UMThunkStub_LOCALVARS               (3*4)   // UMEntryThunk*, Thread*, dummy void*
```

Comment:
A nit - I would keep this unchanged instead of including the alignment in the local variables and modify the UMThunkStub_FIXEDALLOCSIZE definition to include the +4 alignment. Having the alignment a part of the local vars seems a little bit odd to me.